### ISSUE TITLE
Augmentation is not applied in trainnig 

### ISSUE THREAD

**Issue by rokty** [ultralytics/ultralytics#7001]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Hi, i installed yolo with pip install and didn't clone the repository.
I am trying to train a  **classification** model with the following code in ipynb:
`datafolder=xxx`
`results = model.train(data=datafolder, epochs=100, imgsz=800,degrees=20, scale=0.0, shear=0.2, perspective=0.2, flipud=0.5, fliplr=0.5, mosaic=0,augment=True, verbose=True)`

In the output text when the training start, it also shows that the augmentation parameters are the same as my input, so it's not overruled by other configs.

However I noticed the accuracy for each epoch is the same in different runs. Say in the end of epoch 1 , the accuracy is always 0.757 , and it's 0.875 in epoch 2 for all runs. Thus i checked the 'train_batch#.jpg' in the train# folder and found that there is no augmentation at all.  All the input images are just like the original ones.

That was before I intsalled albumentations. After i installed albumentations there are some agmentations, like flip or scale, which is showed in the training output and train_batch#.jpg. But there are still no rotation or other augmentations which should be in yolo base code.

And after I uninstalled albumentations, there is still no augmentation at all. Is there anyway I can make the intrinsic augmentation of yolo to work?

**Comment by glenn-jocher** [comment]
@rokty hi there! It's unusual for the augmentations not to be applied, especially if they're specified in your training command and reflected in the training output. YOLOv8's native augmentations should work independently of external libraries like `albumentations`.

Here are a few steps you might consider:
- Ensure you're using the latest version of the Ultralytics YOLOv8 package.
- Verify that your dataset is correctly formatted; augmentation might fail if the data isn't as expected.
- Check the remainder of your training log for any warnings or errors related to data loading or augmentations.

If you've confirmed these points and the issue persists, please open an issue in the Ultralytics YOLOv8 repo with details about your setup like the operating system, Python version, and the exact output from the training process. We'll look into it promptly!

Remember to include all relevant details. Your thorough description helps us provide effective support. Thanks for contributing to the YOLOv8 community! 😊
